src/main/java/net/sf/jabref/gui/importer/ImportCustomizationDialog.java

```diff
@@ -79,21 +78,19 @@ public ImportCustomizationDialog(final JabRefFrame frame) {
 
         JButton addFromFolderButton = new JButton(Localization.lang("Add from folder"));
         addFromFolderButton.addActionListener(e -> {
-            CustomImporter importer = new CustomImporter();
 
             FileDialog dialog = new FileDialog(frame).withExtension(FileExtensions.CLASS);
             dialog.setDefaultExtension(FileExtensions.CLASS);
             Optional<Path> selectedFile = dialog.showDialogAndGetSelectedFile();
 
             if (selectedFile.isPresent() && (selectedFile.get().getParent() != null)) {
```

Comment:
I think there it would actually make sense to use ifPresent or is there somehow a possible exception thrown?